src/views/dashboard/playground/docs/getting-started.md

```diff
@@ -172,51 +114,151 @@ Multiple fields can be grouped together. The following SQL statement returns the
+as specified by the `LINEAR` fill option.
+
+```sql (line|usage_user_5m_avg|ts|hostname)
+SELECT 
+    ts,
+    hostname,
+    avg(usage_user) RANGE '5m' FILL LINEAR as usage_user_5m_avg
+FROM cpu_metrics 
+ALIGN '1m' BY (hostname) ORDER BY ts DESC LIMIT 100;
+```
+
+There are other fill options available, please refer to [FILL OPTIONS](https://docs.greptime.com/reference/sql/range#fill-option) documentation.
+
+### Write query results to a new table
+
+After obtaining the query results, you can store them in a new table using the `INSERT INTO` statement.
+Before storing the query results in a new table, you need to create the table first.
+
+:::tip NOTE
+GreptimeDB supports other protocols that offer a schemaless approach to writing data that eliminates the need to manually create tables. See [Automatic Schema Generation](https://docs.greptime.com/user-guide/write-data/overview#automatic-schema-generation).
+:::
+
+The following SQL statement creates a new table called `cpu_metrics_avg_user` to store the average CPU usage of each host at a 5-minute interval.
 
 ```sql
-select date_trunc('second', ts) as dt,
-    avg(usage_user),
-    avg(usage_system),
-    avg(usage_idle)
-    FROM cpu_metrics
-    GROUP BY dt
+CREATE TABLE IF NOT EXISTS cpu_metrics_avg_user (
+    hostname STRING,
+    usage_user_5m_avg DOUBLE,
+    ts TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    TIME INDEX(ts),
+    PRIMARY KEY(hostname)
+);
 ```
 
-See more about [`date_trunc` function](https://docs.greptime.com/reference/sql/functions#date-trunc).
+Then, you can write the query results to the new table by using the `INSERT INTO` statement.
 
-<!-- TODO: The following SQL statement returns the average CPU usage of all hosts every 5 minutes.-->
+```sql
+INSERT INTO cpu_metrics_avg_user(ts, hostname, usage_user_5m_avg)
+    SELECT 
+        ts,
+        hostname,
+        avg(usage_user) RANGE '5m' FILL LINEAR as usage_user_5m_avg
+    FROM cpu_metrics 
+    ALIGN '1m' BY (hostname);
+```
 
-### More aggregate functions
+Check the data in the new table by running the following SQL statement.
 
-There are more aggregate functions available, change `avg` to any of these below
-and try out:
+```sql
+SELECT * FROM cpu_metrics_avg_user LIMIT 100;
+```
+
+Please refer to [INSERT INTO SELECT](https://docs.greptime.com/reference/sql/insert#insert-into-select-statement) documentation for more information.
+
+### Time and date functions
+
+GreptimeDB supports most of the time and date functions in PostgreSQL and Datafusion to help you manipulate time and date conveniently.
+
+For example, you can get the current timestamp by using the `now()` function:
+
+```sql
+SELECT now();
+```
 
-`max` / `min` / `sum`
+Get the timestamp of 5 minites ago by using the `now()` function and `interval` keyword:
+
+```sql
+SELECT now() - interval '5 minutes';
+```
+
+Extract the year from a timestamp by using the `date_part()` function:
+
+```sql
+SELECT date_part('year', now());
```

Comment:
Can we use this in an example of `group by`?

Reply:
🤔 Range Query is a better choice.